socorro/siglists/__init__.py

```diff
@@ -0,0 +1,54 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+
+
+_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+
+
+# This is a hack because sentinels can be a tuple, with the second item being
+# a function to verify if the sentinel applies. It's quite hard to express
+# that in a .txt file, so this special value is here. This list should not
+# grow much, and if it does, we should find a better solution for handling
+# these special values.
+_SPECIAL_EXTENDED_VALUES = {
+    'signature_sentinels': (
+        (
+            'mozilla::ipc::RPCChannel::Call(IPC::Message*, IPC::Message*)',
+            lambda x: (
+                'CrashReporter::CreatePairedMinidumps(void*, '
+                'unsigned long, nsAString_internal*, nsILocalFile**, '
+                'nsILocalFile**)'
+            ) in x
+        ),
+    ),
+}
```

Comment:
I really have no idea how to express this in a text file. And I think it's asking for trouble. There is only one exception at the moment, I think it's fine to have this hack. I made it clear in the comment that we should figure out something better if that were to grow. But it hasn't changed in years...